src/components/DeleteTableDialog.vue

```diff
@@ -193,25 +194,27 @@ export default Vue.extend({
         workspace,
       } = this;
 
+      function tableName(table: TableRow) {
+        // eslint-disable-next-line no-underscore-dangle
+        return table._id.split('/')[0];
+      }
```

Comment:
Is this what you two were thinking about @waxlamp and @AlmightyYakob ?

Reply:
Looks good to me :+1:

Reply:
Actually, I think `table` should be named `tableRow` or something, since it's not actually a table itself, it's a row of a table. Otherwise it looks good.

Reply:
That makes sense.
See 2f51fba